src/TestSuite/Fixture/FixtureManager.php

```diff
@@ -237,7 +238,7 @@ protected function _setupTable($fixture, $db, array $sources, $drop = true)
         $exists = in_array($table, $sources);
 
         if (($drop && $exists) ||
-            ($exists && !$isFixtureSetup && $fixture->schema() instanceof Table)
+            ($exists && !$isFixtureSetup && $fixture instanceof TableSchemaInterface && $fixture->schema() instanceof Table)
```

Comment:
Given that we're only using a single method here, how is the interface better than checking for the method we want?